content/docs/manuals/unikernel-operations.md

```diff
@@ -1,12 +1,134 @@
+2. A service deployment descriptor 
+3. A worker Node with KVM Runtime enabled
+
+## Packaging your first Unikraft image 
+
+Once you build your first Unikernel using [Unikraft](http://unikraft.org), you can create a tarball of the kernel image and the necessary files.
+
+```bash
+myunikernel.tar.gz
+|-- kernel
+`--files1/
+      |--...
+ `--...
+```
+
+Your unikernel tarball MUST contain a file named `kernel`, which is the unikernel image itself. Additionally, it CAN contain a `files1` folder, which is mounted by Oakestra as kernel filesystem at runtime. This folder contains all the additional files for the unikernel to run.
+
+{{< callout context="tip" title="Looking for an example?" icon="outline/rocket" >}}
+Check out the **Unikraft Nginx** example in the Application Catalogue section of the wiki.
+{{< /callout >}}
+
+Once you've packaged your unikernel, you can upload it to a location accessible to Oakestra, such as a web server or a cloud storage service.
+
+## Creating a Service Deployment Descriptor for your Unikernel
+
+Unikernels can be deployed to Oakestra like any other service. You need to create a service deployment descriptor that describes the unikernel service you want to deploy.
+Here is an example of an Nginx server using Unikraft:
+
+```json {title="unikernel-nginx.json"}
+ {
+   "sla_version": "v2.0",
+   "customerID": "Admin",
+   "applications": [
+ {
+       "applicationID": "",
+       "application_name": "nginx",
+       "application_namespace": "test",
+       "application_desc": "Simple demo of a Unikraft Nginx server",
+       "microservices": [
+ {
+           "microserviceID": "",
+           "microservice_name": "nginx",
+           "microservice_namespace": "nginx",
+           "virtualization": "unikernel",
+           "cmd": [
+             ""
+ ],
+           "memory": 400,
+           "vcpus": 1,
+           "vgpus": 0,
+           "vtpus": 0,
+           "bandwidth_in": 0,
+           "bandwidth_out": 0,
+           "port": "9000:80",
+           "storage": 0,
+           "code": "https://github.com/oakestra/oakestra/releases/download/alpha-v0.4.301/nginx_amd64.tar.gz",
+           "arch": [
+             "amd64"
+ ],
+           "state": "",
+           "added_files": []
+ }
+ ]
+ }
+ ```
+
+ #### What's important to note in this SLA?
+
+- The `virtualization` field is set to `unikernel`
+- The `code` field contains the URL to the unikernel tarball. This URL must be accessible to the Oakestra worker nodes. We're using the Oakestra release page as an example here to host the tarball file.
+- The `arch` field specifies the architecture of the unikernel. In this case, it's `amd64`.
+
+The rest of the fields are similar to the ones used for container services.
+You can then deploy the application as usual using the Oakestra Dashboard, the Oakestra CLI, or the Oakestra APIs.
+
+{{< callout context="caution" title="Caution" icon="outline/alert-triangle" >}}
```

Comment:
I think just `Caution` isn't that concise as a title

Reply:
That being said, if the highlight this requirement at the beginning of the page, it might be a bit superfluous here